chef/data_bags/crowbar/bc-template-nova_dashboard.json

```diff
@@ -63,7 +63,7 @@
     "nova_dashboard": {
       "crowbar-revision": 0,
       "crowbar-applied": false,
-      "schema-revision": 23,
+      "schema-revision": 25,
```

Comment:
It's included here; there are two migrations done. (The idea was to keep them separate because they're about two different things)